### ISSUE TITLE
Feature Request: Add the possibility to have author name with annotation

### ISSUE THREAD

**Issue by TiberiusRegenschein** [cgogolin/penandpdf#14]
When reviewing a PDF with serveral people it is very comvenient that with each annotation the name of the reviewer is displayed. Therefore probably a setting of an author name would be needed in Pen&Pdf.
Great tool by the way

**Comment by cgogolin** [comment]
Yes, that is a good feature and rather easy to implement. I will likely include it in the next release.

**Comment by TiberiusRegenschein** [comment]
Attached an example:

1. First line Pen&PDF entry
2. 2nd line Adobe standard entry. Name at the top (striked through by me) and date at the bottom.

"Seite 7" is "page 7" and standard for clustering comments.

Thanks for implementing!
![annotation_example](https://user-images.githubusercontent.com/10390201/44842922-5dbb1f80-ac47-11e8-8149-4ed6802c98b7.jpg)
